src/components/Pokemon.js

```diff
@@ -0,0 +1,48 @@
+import {useEffect, useState} from "react";
+import axios from "axios";
+import './Pokemon.css'
+
+function Pokemon( {pokemonName} ) {
+
+    const [pokemon, setPokemon] = useState({});
+
+  useEffect(() => {
+
+      async function fetchPokemon() {
+
+          try {
+              const response = await axios.get(`https://pokeapi.co/api/v2/pokemon/${pokemonName}`)
+             /* console.log(response.data);*/
+              setPokemon(response.data);
+          } catch (error) {
+              console.error(error)
+          }
+      }
+
+      if (pokemonName) {
+      fetchPokemon();
+      }
+  }, [pokemonName]) ;
+
+  const {name, moves, weight, abilities, sprites } = pokemon
+
+    return (
+
+            <div className={"pokemon-wrapper"}>
+                {Object.keys(pokemon).length > 0 &&
```

Comment:
Dan kan deze check ook iets simpeler
```suggestion
                {pokemon &&
```